src/DataProvider/OperationDataProviderTrait.php

```diff
@@ -75,7 +76,18 @@ private function getSubresourceData($identifiers, array $attributes, array $cont
             throw new RuntimeException('Subresources not supported');
         }
 
-        return $this->subresourceDataProvider->getSubresource($attributes['resource_class'], $identifiers, $attributes['subresource_context'] + $context, $attributes['subresource_operation_name']);
+        $subresourceIdentifiers = [];
+        foreach ($attributes['subresource_context']['identifiers'] as list($identifier, , $isItem, $identifiedBy)) {
```

Comment:
```suggestion
        foreach ($attributes['subresource_context']['identifiers'] as [$identifier, , $isItem, $identifiedBy]) {
```
I prefer personally.